frontend/vue/components/UserAccount/UserAccountLayout.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="user-account">
+    <nav class="user-account__section-nav">
+      <div class="user-account__section-nav__user-data">
+        <span class="user-account__section-nav__user-data__name">{{ userName }}</span>
+        <span class="user-account__section-nav__user-data__role">{{ userRole }}</span>
+      </div>
+      <div class="user-account__section-nav__link-list">
+        <AppLink
```

Comment:
Can we prevent these from opening in new tabs?

Reply:
it should not open new tabs. I will take a look.